pkg/config/context.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"context"
+	"errors"
+)
+
+type contextKey string
+
+const (
+	currentProfileContextKey = contextKey("__current_profile")
+)
+
+func CreateProfileContext(ctx context.Context, profileName string) context.Context {
```

Comment:
what is that?

Reply:
i need to command current profile in the state for getting the Portal URL

This function save the current profile to the context